commands/index.ts

```diff
@@ -60,20 +72,22 @@ export class Commands {
     return this.commandMap.get(command.trim())
   }
   
-  public async execute(message: string): Promise<boolean> {
-    const [command, query]: (string | undefined)[] = message.trim().split(SPLIT_OPERAND_QUERY_REGEX).slice(1)
+  public async execute(message: string): Promise<CommandResponse> {
+    const split: (string | undefined)[] = message.trim().split(/\s+/)
+    const command = split[0]
+    const query = split.slice(1).join(' ')
 
     if (command == null) {
-      return true;
+      return undefined;
```

Comment:
Huh, this works? TypeScript is odd.

Reply:
This is ugly at the moment because I haven't finished refactoring the return types yet. It will improve.